workers/unity/Packages/com.improbable.gdk.tools/GdkToolsConfigurationWindow.cs

```diff
@@ -137,8 +141,18 @@ private void DrawCodeGenerationOptions()
                 }
 
                 GUILayout.Label(RuntimeIpLabel, EditorStyles.boldLabel);
-                toolsConfig.RuntimeIp = GUILayout.TextField(toolsConfig.RuntimeIp);                    
+                toolsConfig.RuntimeIp = GUILayout.TextField(toolsConfig.RuntimeIp);
 
+                GUILayout.Label(DevAuthTokenSectionLabel, EditorStyles.boldLabel);
+
+                GUILayout.Label(DevAuthTokenDirLabel, EditorStyles.label);
+                toolsConfig.DevAuthTokenDir = GUILayout.TextField(toolsConfig.DevAuthTokenDir);
+
+                GUILayout.Label($"Token filepath: {toolsConfig.DevAuthTokenFilepath}", EditorStyles.helpBox);
+
+                GUILayout.Label(DevAuthTokenLifetimeLabel, EditorStyles.label);
+                toolsConfig.DevAuthTokenLifetimeDays =
+                    EditorGUILayout.IntSlider(toolsConfig.DevAuthTokenLifetimeDays, 1, 90);
```

Comment:
Is this a good and useful time range? Do we really want 3 month tokens?

Reply:
Its the maximum allowed by toolbelt I think

Reply:
yes, you can't have tokens older than 3 months